torch/_torch_docs.py

```diff
@@ -662,6 +707,29 @@ def merge_dicts(*dicts):
     tensor([ 0.9833,  0.0811, -1.9743, -1.4151])
 """.format(**common_args))
 
+add_docstr(torch.atanh,
+           r"""
+atanh(input, out=None) -> Tensor
+
+Returns a new tensor with the inverse hyperbolic tangent of the elements of :attr:`input`.
+All values in the input tensor must be in the open interval `(-1, 1)`.
```

Comment:
Per discussion, make this consistent with the discussion of the domain for acosh. That is, the values don't *have* to be in the domain, but if they're not they're mapped to nan and that's OK.

Reply:
Done, please check the doc language if that can be made simpler. Thanks, Ruberry!